content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/least_packages.rst

```diff
@@ -0,0 +1,77 @@
+======================
+Least packages removal
+======================
+
+The *Least Packages* removal strategy fulfills an order by opening the fewest number of packages,
+which is ideal for maintaining organized stock without needing to open multiple boxes.
+
+To understand how the removal strategy works, consider the following example, featuring a warehouse
+that stores Packages of Flour in bulk.
+
+Purchased products are packaged in: `10 kg`, `30 kg`, and `50 kg`.
+
+Then, the flour is stored in a sealed package in quantities of `100 kg`.
+
+To minimize moisture, and/or prevent pests from entering open packages, the least packages removal
+strategy is used to pick from a single, opened package.
+
+.. _inventory/warehouses_storage/pkg-qty:
+
+.. example::
+   A package of `100 kg` of flour is depleted to `54 kg` after fulfilling some orders. There are
+   other packages of `100 kg` in stock.
+
+   #. When an order for `14 kg` of flour is placed, the package of `54 kg` is selected.
+   #. However, if an order is placed for *more* than `54 kg` of flour, one of the `100 kg` packages
+      is used to fulfill the order.
```

Comment:
Would it pull the entirety of the flour from a new package, or take the remainder of the open package and then the rest from a new package? I think it might be the latter, since that results in only one open package, instead of two.